test/sendEmail.test.js

```diff
@@ -20,7 +37,39 @@ describe('sendTestEmail function', () => {
     const result = await sendTestEmail(recipientEmail);
 
     // Verify the content of the sent email
-    expect(result.envelope.from).toEqual(process.env.EMAIL_USERNAME); // Verify the 'from' address
-    expect(result.envelope.to).toEqual([recipientEmail]); // Verify the 'to' address
+    expect(result.envelope.from).toEqual(process.env.EMAIL_USERNAME);
+    expect(result.envelope.to).toEqual([recipientEmail]);
+  });
+
+  // it('connects to Ethereal SMTP service', async () => {
+  //   const transporter = nodemailer.createTransport({
+  //     host: 'smtp.ethereal.email',
+  //     port: 587,
+  //     auth: {
+  //       user: 'irma.cronin26@ethereal.email',
+  //       pass: 'mDHwkxB1yhxaEn3yCk',
+  //     },
+  //   });
+
+  //   const info = await transporter.verify();
+
+  //   expect(info).toBeDefined();
+  //   expect(info.accepted.length).toBeGreaterThan(0);
+  // });
```

Comment:
Cool test! We can keep or if it's not doing what you expect, I'd be fine with removing this before we merge.

Reply:
I have deleted this test for now, anyway if we need later, we can add,  commit 299ecf3bf470778ca6c1c2880f1d126182f6e002